src/NosCore.GameObject/Networking/ServerManager.cs

```diff
@@ -12,17 +13,28 @@ namespace NosCore.GameObject.Networking
 {
     public sealed class ServerManager : BroadcastableBase
     {
+        private long _lastGroupId = 1;
+
         private static ServerManager _instance;
         private ServerManager()
         {
+            Groups = new ConcurrentDictionary<long, Group>();
```

Comment:
just on the property. let's avoid adding new lines on servermanager when you can